lbrynet/lbrynet_daemon/Daemon.py

```diff
@@ -1587,9 +1573,13 @@ def jsonrpc_get(self, uri, file_name=None, timeout=None, download_directory=None
         timeout = timeout if timeout is not None else self.download_timeout
         download_directory = download_directory or self.download_directory
 
-        resolved = yield self.session.wallet.resolve_uri(uri)
+        resolved_result = yield self.session.wallet.resolve(True, 0, 10, uri)
```

Comment:
another place where the `page` and `page_size` could be avoided if they are made defaults. DRY, maaaaan :-)